models/git/branch.go

```diff
@@ -378,3 +380,22 @@ func RenameBranch(ctx context.Context, repo *repo_model.Repository, from, to str
 
 	return committer.Commit()
 }
+
+// FindRecentlyPushedNewBranches return at most 2 new branches pushed by the user in 6 hours which has no opened PRs created
```

Comment:
What if

- I pushed branch `feature/x` to repo
- I saw the flash message and clicked it to open a PR, so convenient.
- Someone reviewed the PR and pointed it out that the idea was wrong, so I closed the PR and gave up.
- And could I see the flash message again, right? But I had already created a PR through it.

Reply:
And please update the description of this PR, it's not "3 days" any longer.

Reply:
> And please update the description of this PR, it's not "3 days" any longer.

Updated.